evap/staff/templates/staff_evaluation_person_management.html

```diff
@@ -20,15 +20,17 @@ <h6 class="card-subtitle mb-2 text-muted">{% trans 'From Excel file' %}</h6>
                             <button type="button" class="btn-link" onclick="copyHeaders(['Title', 'First name', 'Last name', 'Email'])">{% trans 'Copy headers to clipboard' %}</button>).
                             {% trans 'This will create all containing users.' %}
                         </p>
-                        {% include 'bootstrap_form.html' with form=participant_excel_form wide=True %}
+            	        {% include 'bootstrap_form.html' with form=participant_excel_form wide=True %}                    
                     </div>
-                    <div class="card-footer text-center">
+                    <div class="card-footer text-center">                      
```

Comment:
please pay attention to the leading and trailing whitespace changes here and above